release/nrc/nrc.str.sencoten/source/nrc.str.sencoten.model.kps

```diff
@@ -9,7 +9,7 @@
   </Options>
   <Info>
     <Name URL="">SENĆOŦEN (Saanich Dialect) Lexical Model</Name>
-    <Copyright URL="">© 2019 National Research Council Canada</Copyright>
+    <Copyright URL="">© 2019 Timothy Montler and the W̱SÁNEĆ School Board</Copyright>
     <Author URL="mailto:Eddie.Santos@nrc-cnrc.gc.ca">Eddie Antonio Santos</Author>
     <Version>1.0.3</Version>
```

Comment:
I think this should be version `1.0.4` if you want the package to be rebuilt on the CI.

Reply:
Okay, done!